tests/SarTests.cpp

```diff
@@ -114,3 +114,16 @@ TEST_F(SarSensorModel, imageToRemoteImagingLocus) {
       EXPECT_NEAR(locus.direction.y, -1.0, 1e-8);
       EXPECT_NEAR(locus.direction.z, 0.0, 1e-8);
 }
+
+TEST_F(SarSensorModel, adjustedPositionVelocity) {
+  std::vector<double> adjustments = {1000000.0, 0.2, -10.0, -20.0, 0.5, 2000000.0};
+  csm::EcefCoord adjPosition = sensorModel->getAdjustedSensorPosition(0.0, adjustments);
+  csm::EcefVector adjVelocity = sensorModel->getAdjustedSensorVelocity(0.0, adjustments);
+  
+  EXPECT_NEAR(adjPosition.x, 4737399.9991294323, 1e-2);
+  EXPECT_NEAR(adjPosition.y, 0.2, 1e-2);
+  EXPECT_NEAR(adjPosition.z, -18.0667951486008143, 1e-2);
+  EXPECT_NEAR(adjVelocity.x, -28.0667951486008143, 1e-8);
+  EXPECT_NEAR(adjVelocity.y, 0.5, 1e-8);
+  EXPECT_NEAR(adjVelocity.z, -1737399.9991294323, 1e-2);
```

Comment:
Don't hard code the values here. Use the un-adjusted position and velocity to get the comparison values.

Reply:
Can do. That's what I did by-hand and you're right that it makes more sense to just do it in here.